packages/web3-net/src/types.ts

```diff
@@ -0,0 +1,9 @@
+/* for some reason typescript is thrown error  src/types.ts(1,9): error TS2305: Module '"web3-utils"' has no exported member 'HexString'. */
+// TODO: use HexString from "web3-utils" => import {HexString} from "web3-utils";
+type HexString = string;
+
+export type NetRPCApi = {
```

Comment:
This file should be renamed to `web3_net_api.ts` as well as this type being named `Web3NetAPI` to match other specs like [Web3EthExecutionAPI](https://github.com/ChainSafe/web3.js/blob/ea41c556130558b37305767a39f25b4306dc0e50/packages/web3-eth/src/web3_eth_execution_api.ts#L19)